prog/vnet/update_load_balancer_node.rb

```diff
@@ -29,6 +41,68 @@ def before_run
     pop "load balancer is updated"
   end
 
+  label def put_certificate
+    cert = load_balancer.active_cert
+    cert_payload = cert.cert
+    cert_key_payload = OpenSSL::PKey::RSA.new(cert.csr_key).to_pem
+    vm.vm_host.sshable.cmd("sudo -u #{vm.inhost_name} mkdir -p #{cert_folder}")
+    vm.vm_host.sshable.cmd("sudo -u #{vm.inhost_name} tee #{cert_path}", stdin: cert_payload)
+    vm.vm_host.sshable.cmd("sudo -u #{vm.inhost_name} tee #{key_path}", stdin: cert_key_payload)
+    hop_start_certificate_server
+  end
+
+  label def start_certificate_server
+    service_name = "#{vm.inhost_name}_cert_server"
+    service_file_path = "/vm/#{vm.inhost_name}/#{service_name}.service"
+    systemd_service = <<SYSTEMD
+[Unit]
+Description=Certificate Server
+After=network.target
+
+[Service]
+NetworkNamespacePath=/var/run/netns/#{vm.inhost_name}
+ExecStart=busybox httpd -f -p [FD00:0B1C:100D:CE::]:8080 -h #{cert_folder}
+Restart=always
+Type=simple
+ProtectSystem=strict
+PrivateDevices=yes
+PrivateTmp=yes
+ProtectHome=yes
+ProtectKernelModules=yes
+ProtectKernelTunables=yes
+ProtectControlGroups=yes
+NoNewPrivileges=yes
+ReadOnlyPaths=#{cert_path} #{key_path}
+User=#{vm.inhost_name}
+Group=#{vm.inhost_name}
+SYSTEMD
+
+    vm.vm_host.sshable.cmd("sudo -u #{vm.inhost_name} tee #{service_file_path}", stdin: systemd_service)
+
+    start_server_cmds = <<CMD
+systemctl enable #{service_file_path}
```

Comment:
I know you don't like putting code to the dataplane. But I just want to say this changes remind me the `rhizome/host/lib/spdk_setup.rb` file. We pass secrets via stdin, then create service files, enable/start services, then delete them while removing etc.